docker-compose.yml

```diff
@@ -118,5 +114,28 @@ services:
       - initialize-ssi-bridge
       - traefik
 
+  dashboard:
+    build:
+      context: ./dashboard
+    container_name: "is-dashboard"
+    restart: always
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.dashboard.entrypoints=web"
+      - "traefik.http.routers.dashboard.rule=PathPrefix(`/`)"
+      - "traefik.http.services.dashboard.loadbalancer.server.port=3055"
+    environment:
+      - VITE_IOTA_IS_SDK_API_KEY=${API_KEY}
+      - VITE_IOTA_IS_SDK_GATEWAY_URL=http://0.0.0.0:3000
+    depends_on:
+      - ssi-bridge
+      - audit-trail-gw
+    ports:
```

Comment:
Remove the "port" section: it will be exposed by traefik

Reply:
this will avoid the constant refreshing when using `npm run dev`  see https://github.com/sveltejs/kit/issues/1468